4 json and json ld documentation

haydenkarl22 opened the pull request:
Fixes issue #4: Added JSON, JSON-LD, and JSON-LD @context handle documentation to JSON documentation READ ME file.

kate-holdener commented on .github/CODEOWNERS:
```diff
@@ -0,0 +1 @@
+* @jacksayshi @alar12
```
This doesn't seem like the change your PR should be making

haydenkarl22 replied:
I had to make this change to fix merge conflicts with the repo. We attempted walking back my branch and then pulling the repo again, but it would never pull the CODEOWNERS file, and since it wouldn't, I could never push my code to the repo. We fixed it by making my own CODEOWNERS file just as it was in the repo. Sorry for not clarifying this when I first created this pull request.

jacksayshi reviewed (changes requested):
Thank you for working on this @haydenkarl22. The documentation you have generated is really good. You made it very clear and easy to understand. Great idea to include more resources at the end.

Here are few suggestions that could help us :

- A website called [json-ld playground](https://json-ld.org/playground/) that helps users understand how they work by comparing with the expanded form. If you think new users can benefit from that feel free to include it along with other resources.

- Also I think it would be easier for users to navigate to pages if you could make the actual external resource links as hyperlinks.

haydenkarl22 commented:
Added 

> Thank you for working on this @haydenkarl22. The documentation you have generated is really good. You made it very clear and easy to understand. Great idea to include more resources at the end.
> 
> Here are few suggestions that could help us :
> 
> * A website called [json-ld playground](https://json-ld.org/playground/) that helps users understand how they work by comparing with the expanded form. If you think new users can benefit from that feel free to include it along with other resources.
> * Also I think it would be easier for users to navigate to pages if you could make the actual external resource links as hyperlinks.

I've considered the suggestions, and I agree they are good ideas, so I added them to the markdown document. The JSON-LD playground webpage is a link in the documentation, and all the links are now hyperlinks. I've also added padding to the text to make it easier to digest. This issue is ready for final review.

alar12 reviewed (approved):
Good job! The content explains JSON, JSON-LD, and @context clearly. Adding JSON-LD document is helpful. The developers/users will get to know how to develop/use rerum with JSON objects.

jacksayshi reviewed (approved):
Thank you for making the changes. It looks good.
